### ISSUE TITLE
Search box API call for search results view

### ISSUE THREAD

**Issue by nikomakr** [nikomakr/studyingShowTime#75]
**Description / User Story:**

Further develop and optimised search button. Develop from scratch functionality of search box.
Utilise both APIs to retrieve data, get value written by user in search box and treat it as keyword for the API, show APIs results in console by linking calls in order. Block blank search.

**Acceptance Criteria:**

- Blocks blank search, so no input on search box and user clicks search. Expected no redirection on searchPage
- Search button to stop just redirecting to searchpage without not calling both APIs
- Link both APIs, so we get the value input by user in search box, call moviesminidatabase.p.rapidapi.com fetch the data and subject to primary key to call the imdb188.p.rapidapi.com API and and get data results
- store data on local storage

**QA (quality assurance) / testing:**

Must meet requirements addressed in acceptance criteria

**Comment by nikomakr** [comment]
@clamadi for your convenience and to save some API calls, so we won't run out of calls I recorded my testing. FYI in my testing in video below I blocked Shola's js script so I don't have any noise. That means I block line 27 & 48 from script.js as it is not relevant to this work which sits on search.js & result.js files. 

[ ](https://www.awesomescreenshot.com/video/23354375?key=e493e91d3cc5c39a0ef6fbf49a156c3c)
